.env.example

```diff
@@ -1,8 +1,8 @@
 protocol=http # http or https
 domain-a=localhost # domain-aaa.com
-domain-a-background=bg-green-100
+domain-a-background=bg-sky-50
```

Comment:
Reason I selected green is to provide a clear context that green means allowed and anything appearing green corresponds to what should be allowed in the page

Reply:
@gagan0123 the `sky-50` is close to the blue background color on Privacy Sandbox website

Reply:
Perfect I have reverted it